client/js/screens.js

```diff
@@ -91,19 +91,34 @@ var ingame = {
     init: function() {        
         game.stage.backgroundColor = '#454645';
         
+        map = game.add.tilemap('map');
```

Comment:
encapsulate map manipulation and creation in an object

Reply:
A bit obsessed with putting things into objects, so at your own discretion: some ideas for thought
Wrap 'game' into a Game class so we can maintain singleton objects of stuff like map, screen, state, etc, all neatly tucked into one place, and easily manage dependencies between doing things in wrapper methods; initialization of stuff could all be handled in one place as well. The same idea can be applied to stuff like map, player, etc.

Reply:
Also, if we have a 'Screen' object each Screen can have a collection of text objects, and this would couple text with the screen we are currently on. Background color and other things can also be set - in more advanced cases we can use this to create pop-up windows or mouse-over text pop-ups, but that is just for speculation.